order/interface.go

```diff
@@ -570,6 +589,25 @@ func (b *Bid) Digest() ([sha256.Size]byte, error) {
 			return result, err
 		}
 
+	case VersionSidecarChannel:
+		var buf bytes.Buffer
+		if b.SidecarTicket != nil {
+			err := sidecar.SerializeTicket(&buf, b.SidecarTicket)
```

Comment:
nit: can also just write directly to `msg` if done at the end

Reply:
Good idea, done.